Bottom: e9886d09a7f4d1bc5dc20ae96aa6d25126e80adf
Top:    96001da527b0067a0bef796dbb448b8e01f95a3e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:37:56 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index aa6d1c7..5e54508 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1595,6 +1595,12 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+#if 1
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+#endif
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
